backend/coreapp/compilers.py

```diff
@@ -256,6 +276,27 @@ def preset_from_name(name: str) -> Optional[Preset]:
     cc='cpp -P "$INPUT" | ${WINE} "${COMPILER_DIR}"/cc1n64.exe -quiet -G0 -mcpu=vr4300 -mips3 -mhard-float -meb ${COMPILER_FLAGS} -o "$OUTPUT".s && ${WINE} "${COMPILER_DIR}"/asn64.exe -q -G0 "$OUTPUT".s -o "$OUTPUT".obj && "${COMPILER_DIR}"/psyq-obj-parser "$OUTPUT".obj -o "$OUTPUT" -b -n',
 )
 
+# MACOS9
+MWCPPC_CC = 'printf "%s" "${COMPILER_FLAGS}" | xargs -x -- ${WINE} "${COMPILER_DIR}/MWCPPC.exe" -o object.o "${INPUT}"'
+MWCPPC_AS = 'printf "%s" "-dis -h -module ".${FUNCTION}" -nonames -nodata" | xargs -x -- ${WINE} "${COMPILER_DIR}/MWLinkPPC.exe" "${INPUT}" > ${COMPILER_DIR}/code.s'
+MWCPPC_CONVERT_GAS_SYNTAX = 'python3 ${COMPILER_DIR}/convert_gas_syntax.py "${COMPILER_DIR}/code.s" ".${FUNCTION}" > "${INPUT}"; rm ${COMPILER_DIR}/code.s; powerpc-linux-gnu-as "${INPUT}" -o "${OUTPUT}"'
```

Comment:
these should be able to be piped, `;`d, and/or `&&`d together and put all into the compiler flags